uxarray/grid/grid.py

```diff
@@ -2485,3 +2498,89 @@ def get_faces_between_latitudes(self, lats: Tuple[float, float]):
 
         """
         return faces_within_lat_bounds(lats, self.face_bounds_lat.values)
+
+    def get_faces_containing_point(self, point, tolerance=ERROR_TOLERANCE):
+        """Identifies the indices of faces that contain a given point. This function will typically return a
+        single face, unless the point falls directly on a corner or edge, where there will instead be multiple values.
```

Comment:
"This function will typically return a single face, unless the point falls directly on a corner or edge, where there will instead be multiple values."

Can we move this inside of the description of the `index` parameter?